### DISCUSSION TITLE
Dropdown API load on page load.

### DISCUSSION THREAD

**Discussion by vidoardes** [fomantic/Fomantic-UI#1042] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted

## Problem
Hi there, just looking at moving over from Semantic UI and I didn't know if changes over here would help solve a problem I've had. I have a drop down that gets it's data from an API call, which works perfectly, except the list isn't loaded until the dropdown is clicked. I cannot for the life of me figure out how to get it to make the call when the page first loads so it is pre-poulated.

Is this even possible? Any advice would be appreciated.

**Comment by prudho** [comment, marked as answer]
I'm not sure this is doable actually... Maybe you can call `$('.ui.dropdown').dropdown('queryRemote', '', function() {});` so it should preload your data, but i'm not confident about this...

**Comment by lubber-de** [comment]
Closing because prudho's advise works as expected with the current codebase and there was no more response from the issuer.
The poposed solution can be tried here http://jsfiddle.net/4tacdvLq/
